### PR TITLE
Fix navbar, fix footer colors

### PR DESCRIPTION
**Pull request by kevin-zou**
## Description
Couple of fixes:
1. Nav bar wasn't listening to scroll events any more and was not fading. This was because I thought I could remove the parentheses on the function call but turns out you can't :')
2. The text in the footer was the wrong color. It should have be white text that is green on highlight, so I just swapped that around

### CODE REVIEW

File: components/NavBar.js [hunk 1]
```diff
@@ -214,7 +214,7 @@ const NavBar = () => {
   }
 
   useEffect(() => {
-    window.addEventListener('scroll', handleScroll);
+    window.addEventListener('scroll', handleScroll());
     window.addEventListener('resize', handleResize);
```
Comment (vincentchiang1) [inline components/NavBar.js:218]: would we need to add parentheses to handleResize as well for the event listener on line 218 to work?

**Review by vincentchiang1 — APPROVED** [review]
aside from my question with the listener, the fix looks great :)

File: components/NavBar.js (hunk 1 above)
Comment (kevin-zou) [reply components/NavBar.js:218]: Hmmm it seems to work without it 🤔 

`handleScroll` is a little different in that it returns another function, whereas `handleResize` is the actual function itself, so maybe that has something to do with it? I'm not completely sure either way though lol

Comment (vincentchiang1) [reply components/NavBar.js:218]: oooh i didn't look into it, it makes sense if it works without it since you just said they're different. sounds g!

**Pull request merged**